src/main/java/com/github/invictum/dto/AbstractDto.java

```diff
@@ -20,12 +20,28 @@
 
     private String extractData(Object object, Field attribute) {
         attribute.setAccessible(true);
+        Object value;
         try {
-            return (String) attribute.get(object);
-        } catch (IllegalAccessException e) {
+            value = attribute.get(object);
+        } catch (IllegalAccessException var4) {
             LOG.error("Failed to get data for {} attribute", attribute);
+            return null;
         }
-        return null;
+
```

Comment:
1) I think it is acceptable to use several returns here.
2) Use if ... else clause. In last else use some default convert method (e. g. value.toString()). Log default converting usage.